r/tests/testthat/test-dplyr-string-functions.R

```diff
@@ -240,11 +255,12 @@ test_that("str_replace and str_replace_all", {
 })
 
 test_that("strsplit and str_split", {
+  
   df <- tibble(x = c("Foo and bar", "baz and qux and quux"))
   
   expect_dplyr_equal(
     input %>%
-      transmute(x = strsplit(x, "and")) %>%
+      mutate(x = strsplit(x, "and")) %>%
```

Comment:
`mutate()` and `transmute()` would do exactly the same thing in these tests, right? (Because there's only one column in the data, and it gets replaced.)

Reply:
It didn't work for me previously and I wasn't sure why, but now I think it was triggered by something else.

Reply:
Ok, cool, no need to change anything 👍